src/main/java/service/impl/FruitShopServiceImpl.java

```diff
@@ -0,0 +1,53 @@
+package service.impl;
+
+import static service.OperationType.BALANCE;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import model.FruitTransaction;
+import service.FruitShopService;
+import service.OperationType;
+import storage.ParserService;
+
+public class FruitShopServiceImpl implements FruitShopService {
+    private final ParserService parserServiceImpl;
+
+    public FruitShopServiceImpl(ParserService parserServiceImpl) {
+        this.parserServiceImpl = parserServiceImpl;
+    }
+
+    @Override
+    public List<FruitTransaction> processingData(List<FruitTransaction> parsedInfo) {
```

Comment:
remove this class completely
instead:
- create a storage class, where you will store fruit and its current quantity
- create a transaction processor class, which will call strategy for each transaction
- create strategy classes and handler classes for all operations
- handlers should perform changes in storage
- storage should have methods to add or subtract quantity for given fruit
- storage should have `getEntries()` method which will return entry set with all info about fruits and theirs quantities